bin/better-unit

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const betterUnit = require('../index')
+const minimist = require('minimist')
+
+// Parse options
+const options = minimist(process.argv.slice(2), {
+  alias: {
+    recursive: 'r',
+    testspec: 't'
+  }
+})
+const files = options._
+delete options._
+delete options.r
+delete options.t
+
+// Runt tests
+betterUnit.run(files, options)
+  .catch((error) => {
+    console.error(error.stack)
+    process.exitCode = 1
+  })
```

Comment:
It seems pretty clear to me: https://github.com/clipper-digital/better-unit/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R36

We can't do a simple pass through without more logic, especially for the reporter options. I honestly don't care about supporting everything.